src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands
+{
+    public class MeetingInvitationHelper
+    {
+        public static string GenerateMeetingTitle(Invitation invitation) 
+            => $"Invitation to Punch Out, IPO-{invitation.Id}, Project: {invitation.ProjectName}";
+
+        public static string GenerateMeetingDescription(Invitation invitation, string baseUrl)
+        {
+            var meetingDescription = "<h4>You have been invited to attend a punch round.</h4>";
```

Comment:
Consider use of @ to make a string of the whole email template (which can be moved to a sep file later). Add "tokens" inside the string-template, such as [InvitationTitle], and run a .Replace of known tokens: .Replace("[InvitationTitle]", invitation.Title)

This goes for all the string making html here. I can assist to show what I mean

Reply:
![image](https://user-images.githubusercontent.com/19545528/106004619-ae9efe80-60b3-11eb-9c05-2bfa79da979c.png)

Reply:
var meetingDescription = 
                @"
                        <h4>You have been invited to attend a punch round.</h4>
                        <p>Title: [InvitationTitle]</p>
                        <p>Type: [InvitationType]</p>
                        <p>Location: [Location]</p>
                        <p>Description: [Description]</p>
                        <p>Scope: </p>
                        [McPkgTable]
                        [CommPkgTable]
                        </br><a href='[Url]'>Open invitation for punch out in ProCoSys.</a>
                ";

Reply:
Okay, yes lets talk about how to do it